modules/generator/localentitylimiter/local_entity_limiter.go

```diff
@@ -77,13 +81,47 @@ func New(maxEntityFunc func(tenant string) uint32, tenant string, limitLogger *t
 		metricMaxActiveEntities:    metrics.maxActiveEntities.WithLabelValues(tenant),
 		metricTotalEntitiesAdded:   metrics.totalEntitiesAdded.WithLabelValues(tenant),
 		metricTotalEntitiesRemoved: metrics.totalEntitiesRemoved.WithLabelValues(tenant),
+
+		// The overflow entity implements the otel metric sdk spec for cardinality control.
+		// See https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/metrics/sdk.md#cardinality-limits
+		overflowEntity: labels.FromStrings("otel_metric_overflow", "true"),
```

Comment:
i don't like this name but only b/c the span metrics connector has drifted from tempo metrics over the years. it's almost like i'd rather have it clearly different since everything else is. wdyt?

we've discussed having an optional "otel compatibility mode" for the metrics generator where the generator would produce otel collector compatible metrics where we could set this to the otel name.

Reply:
Yeah I also don't like this name, any suggestions on what would fit better? What I'd really like is to have something like `service="*"` for span metrics, and something different for service graphs, but I haven't found the right way to make that clean.